lib/pages/settings.dart

```diff
@@ -127,7 +128,7 @@ class AccountsConfigPage extends HookWidget {
             ),
           ) ??
           false) {
-        accountsStore.removeInstance(instanceHost);
+        unawaited(accountsStore.removeInstance(instanceHost));
```

Comment:
Why is it unawaited?

Reply:
cuz it was before, just on the store level

Reply:
Before the actions were synchronous and ignored the save, now that it doesn't ignore it it should be correctly awaited

Reply:
I thought that was the point of this change, to make sure we await for those